tests/test_dataframe.py

```diff
@@ -139,9 +139,15 @@ def test_dataframe_animate(transient_rst):
 
 def test_dataframe_repr(df):
     ref = (
-        "DataFrame<index=MultiIndex<[MeshIndex<name=\"node_ids\", dtype=<class 'int'>>, "
-        "Index<name=\"components\", dtype=<class 'str'>>]>, columns=MultiIndex<[ResultIndex<['U']>, "  # noqa: E501
-        "SetIndex<values=[1]>]>>"
+        "DataFrame<"
+        "index=MultiIndex<["
+        "MeshIndex<name=\"node_ids\", dtype=<class 'int'>>, "
+        "Index<name=\"components\", dtype=<class 'str'>>"
+        "]>, "
+        "columns=MultiIndex<["
+        "ResultIndex<['U (m)']>, "
+        "SetIndex<values=[1]>"
+        "]>>"
     )
     assert repr(df) == ref
```

Comment:
You might want to use this pytest plugin to automatize "golden" tests

https://github.com/oprypin/pytest-golden#pytest-golden

Feel free to ping me about this.

Reply:
oh nice, thx @germa89, I did not know this plugin!

Reply:
Do you know if it works well for numerical values?